packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -50,19 +50,58 @@ def vip_app(container: Container, network: Network, host: str, vip: str):
         return get_test_app(
             network=network,
             host_constraint=host,
+            healthcheck_protocol=Healthcheck.MESOS_HTTP,
             vip=vip,
             container_type=container)
     elif network == Network.USER:
         return get_test_app(
             network=network,
             host_port=unused_port(Network.USER),
+            healthcheck_protocol=Healthcheck.MESOS_HTTP,
             host_constraint=host,
             vip=vip,
             container_type=container)
     else:
         raise AssertionError('Unexpected network: {}'.format(network.value))
 
 
+def pod_vip_app(network: Network, host: str, vip: str):
+    ''' Creates a single app in docker container in pod
+    '''
+    container_port = 0
+    if network == Network.USER:
+        container_port = unused_port(Network.USER)
+    test_uuid = uuid.uuid4().hex
+    app = {
+        'id': '/integration-test-{}'.format(test_uuid),
+        'placement': {'acceptedResourceRoles': ['*', 'slave_public']},
+        'containers': [{
+            'name': 'app-{}'.format(test_uuid),
+            'resources': {'cpus': 0.01, 'mem': 8},
+            'image': {'kind': 'DOCKER', 'id': 'debian:jessie'},
+            'exec': {'command': {
+                'shell': '/opt/mesosphere/bin/dcos-shell python '
+                         '/opt/mesosphere/active/dcos-integration-test/util/python_test_server.py {}'.format(
+                             '$ENDPOINT_TEST' if network == Network.HOST else container_port)
+            }},
+            'volumeMounts': [{'name': 'opt', 'mountPath': '/opt/mesosphere'}],
+            'endpoints': [{'name': 'test', 'protocol': ['tcp'], 'hostPort': 0}],
+            'environment': {'DCOS_TEST_UUID': test_uuid}
```

Comment:
You are missing a environment variable:
https://github.com/mesosphere/dcos-test-utils/blob/master/dcos_test_utils/marathon.py#L94